src/validator.js

```diff
@@ -316,17 +308,17 @@ function validateAttributeString(attrStr, options, regxAttrName) {
 
 // const validAttrRegxp = /^[_a-zA-Z][\w\-.:]*$/;
 
-function validateAttrName(attrName, regxAttrName) {
-  // const validAttrRegxp = new RegExp(regxAttrName);
-  return util.doesMatch(attrName, regxAttrName);
+function validateAttrName(attrName, regexName) {
```

Comment:
Since `regexName` is gonna be constant, do we need to pass it?

Reply:
Will do